docs/deployment/containers/kubernetes.rst

```diff
@@ -129,9 +129,9 @@ Here's an example configuration snippet:
       labels:
         app: crate
     spec:
-      # A static IP address is assigned to this service. This IP address is
-      # only reachable from within the Kubernetes cluster.
-      type: ClusterIP
```

Comment:
You will still need the type. What makes a headless service is the `clusterIP: None` thing.

Reply:
ClusterIP is the default type.
In most definition I found it was omitted for headless services.
The comment is definitely wrong

Reply:
Ah yes you're right, just tried it and it works fine w/o `type`